esx_service/utils/auth_data.py

```diff
@@ -576,7 +638,9 @@ def connect(self):
 
         if not need_create_table:
             if os.path.isfile(self.db_path) and self.need_upgrade_db():
-                self.handle_upgrade_db()
+                error = self.handle_upgrade_db()
+                if error:
+                    raise DbConnectionError(self.db_path)
```

Comment:
Could you also add the test for the case that existing DB schema is before 1.1? I want some test cover the error case here.